docs/docs/remote.md

```diff
@@ -36,31 +38,63 @@ where `$BASE_IMAGE` is `capella/base`, `t4c/client/base` or `t4c/client/pure-var
-    $BASE_IMAGE/remote
-```
-
 Replace the followings variables:
 
-- `$BASE_IMAGE` with `capella/base`, `t4c/client/base` or `t4c/client/pure-variants`. Please check the `In a remote container (RDP)` on the individual page of the base image for additional configuration options.
-- `$RDP_EXTERNAL_PORT` to the external port for RDP on your host (usually `3389`)
+- `$BASE_IMAGE` with `capella/base`, `t4c/client/base` or `t4c/client/pure-variants`. Please check the individual section `In a remote container (RDP)` of the individual base image documentation pages for additional configuration options.
 - `$RMT_PASSWORD` is the password for remote connections (for the login via RDP) and has
   to be at least 8 characters long.
 
-After starting the container, you should be able to connect to
-`localhost:$RDP_EXTERNAL_PORT` with your preferred RDP Client.
+<!-- prettier-ignore-start -->
+=== "Connect via RDP"
+
+    The container image contains a `xrdp` server. To use RDP to connect to the container, run the container with the following command:
+
+    ```zsh
+    docker run -d \
+        -p $RDP_EXTERNAL_PORT:3389 \
+        -e CONNECTION_METHOD=xrdp \
+        -e RMT_PASSWORD=$RMT_PASSWORD \
+        $BASE_IMAGE/remote
+    ```
+
+    Replace `$RDP_EXTERNAL_PORT` with the external port that the RDP
+    server should listen on (usually `3389`).
+
+    After starting the container, you should be able to connect to
+    `localhost:$RDP_EXTERNAL_PORT` with your preferred RDP Client.
+
+    For the login use the followings credentials:
+
+    - **Username**: `techuser`
+    - **Password**: `$RMT_PASSWORD`
+
+    The screen size is set every time the connection is established. Depending on your
+    RDP client, you will also be able to set the preferred screen size in the settings.
+
+    By default, Remmina (RDP client for Linux) starts in a tiny window. To fix that, you can
+    easily set "Use client resolution" instead of "Use initial window size" in the remote
+    connection profile.
+
+
+=== "Connect via XPRA"
+
+    The container image contains a `xpra-html5` server. To use XPRA via HTML5 to connect to the container, run the container with the following command:
 
-For the login use the followings credentials:
+    ```zsh
+    docker run -d \
+        -p $XPRA_EXTERNAL_PORT:10000 \
+        -e CONNECTION_METHOD=xpra \
+        -e RMT_PASSWORD=$RMT_PASSWORD \
+        $BASE_IMAGE/remote
+    ```
 
-- **Username**: `techuser`
-- **Password**: `$RMT_PASSWORD`
+    Then, open a browser and connect to:
+    ```
+    http://techuser:${RMT_PASSWORD}@localhost:${XPRA_EXTERNAL_PORT}?floating_menu=0
+    ```
 
-The screen size is set every time the connection is established. Depending on your
-RDP client, you will also be able to set the preferred screen size in the settings.
+    More configuration options can be passed as query parameters.
+    See the [xpra-html5 documentation](https://github.com/Xpra-org/xpra-html5/blob/master/docs/Configuration.md)
```

Comment:
We don't expose the xpra server directly, only the nginx reverse proxy, which has basic authentication. Therefore, hijacking another session is not possible unless the user gains access to the credentials for another session.

The query parameters are parsed in the frontend itself. Blocking these requests via a reverse proxy is therefore not possible, it would require a code modification of the [xpra-html5](https://github.com/Xpra-org/xpra-html5) library. Maintaining a fork is a lot of effort though, so I would not go there.